pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -1133,8 +1134,11 @@ func MakeNextPodFunc(queue SchedulingQueue) func() *framework.QueuedPodInfo {
 				metrics.UnschedulableReason(plugin, podInfo.Pod.Spec.SchedulerName).Dec()
 			}
 			return podInfo
+		} else if err.Error() == queueClosed {
```

Comment:
I was not mean, but usually we judge the error itself, rather than the error messages. For different error may have same messages.

Reply:
ok, understand